tools/clang/unittests/HLSLExec/ExecutionTest.cpp

```diff
@@ -710,6 +720,14 @@ class ExecutionTest {
                                const char *pShaderModelStr, const char *pShader,
                                Ty *pInputDataPairs, unsigned inputDataCount);
 
+  template <typename T, std::size_t N = 4>
+  //typename = std::enable_if_t<std::is_arithmetic_v<T>>>
```

Comment:
Will rectify this commented out code, my intension was to enforce that the T was of an arithmetic type but I was hitting some compilation errors on this when trying to use the template. I'm not sure it was doing what I expected.

Reply:
Is that necessary? Presumably if it wasn't an arithmetic type you'd pretty soon get errors about it.

Reply:
The errors were easier to parse for a human with these asserts. But you're right, you'll still get errors. Opting to remove the std::enable_if's.